### ISSUE TITLE
Contributor Email Sending Logic is Broken

### ISSUE THREAD

**Issue by feeohnah** [studio-cartogram/fondfolio#104]
**TLDR**
If the contributor provides an email, everything works fine. 
If they don’t provide an email then _nothing_ gets sent (to the giver or bcc Fondfolio)
UNLESS they are also signed into Fondfolio (as any user) and then the email for that logged in user is the one that gets the "giver" AND the contributor emails for that contribution, even though that fondfolio is not associated with their account and they did not contribute...

Basically, it can't find the giver’s email unless the contributor provides theirs and if someone is logged in, it uses that email instead

Scenarios to replicate (tested in Chrome & Safari):
❌ = problem

**Not logged in, no contributor email provided:** 
No Giver Email ❌, No Contributor Email ✅, No BCC Fondfolio ❌

**Not logged in, contributor email provided:** 
Yes Giver Email ✅, Yes Contributor Email ✅, Yes BCC Fondfolio ✅

**Logged in as giver, no contributor email provided:** 
Contributor Email Sent to Giver ❌, Yes Giver Email ✅, No Contributor Email ✅, Yes BCC Fondfolio ✅

**Logged in as giver, email provided:** 
Contributor Email Sent to Giver ❌, Yes Giver Email ✅, No Contributor Email ❌, Yes BCC Fondfolio ✅

**Logged in as other user, no contributor email provided:** 
Contributor Email Sent to Logged in user ❌, No Contributor Email ✅, Yes BCC Fondfolio ✅

**Logged in as other user, contributor email provided:** 
Contributor Email Sent to Logged in user ❌, No Contributor Email ❌, Yes BCC Fondfolio ✅

**Comment by feeohnah** [comment]
@cartogram I'm still getting this typeerror when I test locally and don’t provide an email. 

`TypeError: Cannot read property 'email' of null
    at sendContributionCreateEmail (webpack-internal:///./pages/api/email/contribution-create.ts:48:40)`

The giver email still doesn't send unless the controibutor provides an email.
